specs/scheduler/cluster-init/scripts/23-pyxis.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+PYXIS_VERSION=0.19.0
+SHARED_DIR=/sched/pyxis
+
+function link_plugstack() {
```

Comment:
Could we split this into a new file "22-plugstack.sh" or something similar and then have just one 23-pyxis.sh?
Will it work correctly if the 23-pyxis.sh from default runs before this script?

Reply:
yes will try to have a single script per feature which will behave differently based on the instance it runs on

Reply:
Part of the comment was that the file needs to be in the scripts/ directory if it needs to be run :)